vnext/Microsoft.ReactNative/Views/TouchEventHandler.cpp

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+#include "../Pch/pch.h"
```

Comment:
this one feels a little funky, see if we can clean it up

Reply:
done.